.env.example

```
@@ -1,95 +1,169 @@
-TEI_HTTP_PORT=8080
-
-# Server Configuration
+# Crawler MCP Environment Configuration
+GITHUB_TOKEN=your_github_token_here
+# =============================================================================
+# SERVER CONFIGURATION
+# =============================================================================
 SERVER_HOST=0.0.0.0
 SERVER_PORT=8010
-DEBUG=false
+DEBUG=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the GITHUB_TOKEN key

(UnorderedKey)